internal/controller/connected_client_recorder.go

```diff
@@ -96,6 +98,7 @@ func (ibccr *InventoryBasedConnectedClientRecorder) RecordConnectedClient(ctx co
 	hostData := cleanupCanonicalFacts(logger, originalHostData)
 
 	hostData["account"] = string(account)
+	hostData["org_id"] = string(orgID)
```

Comment:
I think we should wait on adding the org_id to the host message that gets sent to inventory.  I do not know if they are ready to consume that yet.  I do not think inventory would reject the message due to the extra data, but its possible.